model/host/db.go

```diff
@@ -63,6 +63,9 @@ var (
 	StartTimeKey               = bsonutil.MustHaveTag(Host{}, "StartTime")
 	TotalCostKey               = bsonutil.MustHaveTag(Host{}, "TotalCost")
 	TotalIdleTimeKey           = bsonutil.MustHaveTag(Host{}, "TotalIdleTime")
+	Container                  = bsonutil.MustHaveTag(Host{}, "Container")
```

Comment:
These should be named like "ContainerKey" to signify that they mean the bson field name, and also to avoid confusion with the field in the host document itself